src/openforms/conf/ci.py

```diff
@@ -18,7 +18,17 @@
     "oidc": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
 }
 
-LOGGING = None  # shut up logging
+# shut up logging
+LOGGING["loggers"].update(
```

Comment:
I hope don't regret this when debugging build failures but we'll find out.

Reply:
it shouldn't and we can enable `DEBUG=1` envvar to have the exception re-raised :)